support/airflow-provider-vdk/setup.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2021 VMware, Inc.
+# SPDX-License-Identifier: Apache-2.0
+from setuptools import setup
+
+__version__ = "0.0.1"
+
+with open("README.md") as fh:
+    long_description = fh.read()
+
+"""
+Perform the package airflow-provider-vdk setup.
+"""
+setup(
+    name="airflow-provider-vdk",
+    version=__version__,
+    description="Airflow provider for Versatile Data Kit.",
+    long_description=long_description,
+    long_description_content_type="text/markdown",
+    entry_points={
+        "apache_airflow_provider": [
+            "provider_info=vdk_airflow.__init__:get_provider_info"
+        ]
+    },
+    license="Apache License 2.0",
+    packages=[
+        "vdk_airflow",
```

Comment:
Not sure if the packages (or the provider itself for that matter) should include the word airflow. No official providers seem to include it https://github.com/apache/airflow/tree/main/airflow/providers.